.env.example

```diff
@@ -104,6 +104,10 @@ export BOOKWAREHOUSE_NAMESPACE=bookwarehouse
 # optional: Whether to deploy traffic split policy or not
 # Default: true
 # export DEPLOY_TRAFFIC_SPLIT=true
+
+# optional: specify the log level for the enovy's
+# Default: debug
```

Comment:
```suggestion
# Default: error
```

Reply:
This is reflecting the default value for the demo script which is `debug`.